Src/CSharpier.Tests/TestFiles/Directives/Directives.cst

```diff
@@ -20,6 +20,10 @@ class ClassName { }
 
 public class ClassName
 {
+    #region Trailing comment does not indent more than it should #endregion
```

Comment:
Does that `#endregion` end that region at the end of that line?
Or if that's meant to be be part of the comment you should reword it.